src/wrappers/wrapper-array.js

```diff
@@ -37,8 +37,9 @@ export default class WrapperArray implements BaseWrapper {
 
     return this.wrappers.every(wrapper => wrapper.contains(selector))
   }
+
   exists (): boolean {
-    return this.wrappers.length > 0
+    return this.wrappers.filter(wrapper => wrapper.exists()).length > 0
```

Comment:
You can use .every instead—
```
return this.wrappers.every(wrapper => wrapper.exists())
```

Reply:
```
[].every(a => a)
// => true
```

Reply:
I think that it is appropriate to return `true` if there is at least one element matching the search condition.

Reply:
IMHO, I think that it is better to use `some`.

```
return this.wrappers.length > 0 && this.wrappers.some(wrapper => wrapper.exists())
```

Reply:
Thank you for reviewing.
I think the following code using `some` is equivalent.

```
return this.wrappers.some(wrapper => wrapper.exists())
```

Reply:
some would return true if only one wrapper.exists() returns true, every returns true only if every Wrapper exists, which is the behavior of other methods like hasStyle

Reply:
Thank you for reply.
I will use `every`.
I will change this.

Reply:
I changed this.